src/proper_statem.erl

```diff
@@ -294,6 +302,12 @@
 -callback next_state(symbolic_state() | dynamic_state(), term(),
 		     symbolic_call()) -> symbolic_state() | dynamic_state().
 
+-callback all_commands(symbolic_state()) -> [symbolic_call()].
```

Comment:
Totally minor comment here, but it feels inconsistent to list the two callbacks in one order and in the exact opposite in the list of optional_callbacks.